binder/results/changing-enviroment/data/K16/1.csv

```diff
@@ -0,0 +1,1244 @@
+update,mean_fitness,min_fitness,max_fitness,inferiority
```

Comment:
Can you add a README.md into results/changing-environment explaining how/when the data was collected, and put it inside of a folder with the short github hash of the version of the software that generated the data?